grails-app/views/purchaseOrderWorkflow/purchaseOrder/enterOrderDetails.gsp

```diff
@@ -82,14 +88,20 @@
                                 </td>
                             </tr>
                             <tr class='prop'>
-                                <td class='name middle'><label><warehouse:message code="order.purchasingOrganization.label"/></label></td>
+                                <td class='name middle'><label><warehouse:message code="order.destinationParty.label"/></label></td>
                                 <td valign='top'
-                                    class='value ${hasErrors(bean:order,field:'purchasingOrganization','errors')}'>
-                                    <g:selectOrganization name="purchasingParty.id"
-                                                          id="purchasingParty.id"
-                                                          roleTypes="[org.pih.warehouse.core.RoleType.ROLE_PURCHASER]"
-                                                          noSelection="['':'']"
-                                                          class="select2" />
+                                    class='value ${hasErrors(bean:order,field:'destinationParty','errors')}'>
+
+                                    <g:if test="${order.id}">
+                                        ${order.destinationParty.name} (${order.destinationParty?.code})
+                                    </g:if>
+                                    <g:else>
+                                        <g:selectOrganization name="destinationParty.id"
+                                                              id="destinationParty.id" value="${order?.destinationParty?.id}"
+                                                              roleTypes="[org.pih.warehouse.core.RoleType.ROLE_BUYER]"
```

Comment:
Shouldn't we use ROLE_PURCHASER that we have added?

Reply:
I removed it because we already had ROLE_BUYER and Kelsey agreed this was ok.

Reply:
Ok, then it LGTM.